README.md

```diff
@@ -38,31 +40,31 @@ For a total of:
 - GKE Autopilot free-tier: **65$/monthly**
 - GKE Autopilot: **135$/monthly**
 
-> NOTE: Elasticsearch integration is not yet finished.
+> TODO: Elasticsearch integration is not yet integrated here.
```

Comment:
we can remove this comment, it is integrated now.

Reply:
Done